Tools/Create-Azure-Sentinel-Solution/input/Solution_PCMaticSuperShield.json

```diff
@@ -0,0 +1,27 @@
+{
+    "Name": "PCMatic SuperShield",
+    "Author": "John Farley - john@pcmatic.com",
+    "Logo": "<img src=\"https://raw.githubusercontent.com/johnpcmatic/Azure-Sentinel/pcm-sentinel/Solutions/PCMatic%20SuperShield/Logo/pcmatic-green-outline.png\" width=\"200px\" height=\"45px\">",
+    "Description": "This PC Matic SuperShield Sentinel solution provides the capability to ingest [PC Matic SuperShield](https://www.pcmatic.com/pro/) process activity events with Microsoft Sentinel. It helps you gain visibility into what applications and processes are being executed on your endpoints, and which of those are blocked by SuperShield.",
+    "Parsers": [
+      "Parsers/SuperShieldActivity.txt"
+    ],
+    "Analytic Rules": [
+      "Analytic Rules/NRTKnownBadProcess.yaml",
+      "Analytic Rules/NRTUnknownProcess.yaml",
+      "Analytic Rules/NRTAllowedProcess.yaml"
+    ],
+    "Playbooks": [
+      "Playbooks/BlockAllowProcess/azureDeploy.json",
+      "Playbooks/RemoveProcess/azureDeploy.json"
+    ],
+    "Workbooks": [
+      "Workbooks/PCMaticIncidentOverview.json"
+    ],
+    "Key Vault": [
```

Comment:
Hey @johnpcmatic, you can't directly add the keyvault file in input file, can you please add the raw link of this file in pre requisites of playbooks if it is used there, thanks.